### PR TITLE
DOC: clarified read_csv example

### PR DESCRIPTION
**Pull request by teej**
* Simple clarification to the read_csv/read_fwf docs so that the example actually loads data into a dataframe.

### CODE REVIEW

**Comment by teej** [comment]
I wanted to move the User Guide reference down to the examples section similar to the [read_html docs](https://github.com/pandas-dev/pandas/blob/main/pandas/io/html.py#L1095-L1098) but couldn't get it to build correctly locally.

File: pandas/io/parsers/readers.py [hunk 1]
```diff
@@ -432,7 +432,7 @@
 
 Examples
 --------
->>> pd.{func_name}('data.csv')  # doctest: +SKIP
+>>> df = pd.{func_name}('data.csv')  # doctest: +SKIP
```
Comment (phofl) [inline pandas/io/parsers/readers.py:435]: What are you trying to do here?

Comment (teej) [reply pandas/io/parsers/readers.py:435]: I’m just trying to make the examples section here correct.

The example as stated reads the csv, loads it into a dataframe, and then immediately throws it away. No one uses read_csv this way.

Comment (phofl) [reply pandas/io/parsers/readers.py:435]: I don't think we are doing this anywhere else explicitly like that

Comment (MarcoGorelli) [reply pandas/io/parsers/readers.py:435]: agreed, this is fine as-is - closing then, but thanks anyway for the suggestion

**Comment by teej** [comment]
This is how it’s done in near every other documentation page for Pandas. I’m disappointed that we can’t merge a disambiguating, net positive change in arguably the most important docs page for pandas.

**Comment by phofl** [comment]
Just an arbitrary page:

https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.sum.html

We are returning the result.

The actual issue here is, that the example is not checked

**Pull request closed without merging**